rentmyjunk/src/App.js

```diff
@@ -83,6 +87,9 @@ export default function App() {
           <Route path="/register">
             <RegisterRoute />
           </Route>
+          <Route path="/registerAccount">
```

Comment:
We already have a route for Register. It's in the lines directly above this one.

Reply:
Yeah thats what I had asked about on my comment, will address all your comments now :)

Reply:
I think he means to delete line 90.